.github/workflows/buildingPullRequest.yml

```diff
@@ -1,12 +1,8 @@
 name: Building Nitrox Pull Request
-
 on: pull_request
-
 jobs:
   build:
     runs-on: windows-latest
-    if: github.event.pull_request.draft == false
```

Comment:
Is this not supported anymore or why removing it?

Reply:
I thought it was a bit sad not to be able to use the CI for the drafts, I think it's nice to be able to have indicators, and have access to the artefacts (for testing purpose)

Reply:
Seems fair 👍